fluentd/run.sh

```diff
@@ -0,0 +1,12 @@
+if [[ $VERBOSE ]]; then
+  set -ex
+else
+  set -e
+fi
+
+mkdir -p /etc/fluent/configs.d/input/docker
+mkdir -p /etc/fluent/configs.d/input/syslog
+
+ruby generate_throttle_configs.rb "$(cat ${THROTTLE_CONF_LOCATION}/settings)"
```

Comment:
I don't like reading in the config as a parameter like this. I think ruby should just load yaml from `#{ENV['THROTTLE_CONF_LOCATION']}`